src/components/EnlargedPreview/index.js

```diff
@@ -28,7 +30,7 @@ class EnlargedPreview extends Component<Props, State> {
 
   componentWillUpdate(nextProps: Props, nextState: State) {
     if (nextState.isExpanded !== this.state.isExpanded) {
-      this.props.useFullScreen(nextState.isExpanded)
+      this.props.setFullScreen(nextState.isExpanded)
     }
   }
```

Comment:
I think you should add a `componentWillUnmount() { this.props.useFullScreen(false)}` as you always want to remove the full screen here

Reply:
Probably it does need that, but I think it would be out of the scope of this ticket, as it would be changing behavior

Reply:
I've opened a ticket for this